### PR TITLE
zh: add warning to cluster deletion

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/destroy-a-tidb-cluster.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ helm delete ${cluster_name} --purge
 kubectl delete pvc -n ${namespace} -l app.kubernetes.io/instance=${cluster_name},app.kubernetes.io/managed-by=tidb-operator
 ```
 
+> **警告：**
+>
+> 为了确保数据安全，在任何情况下都不要删除 PV，除非您熟悉 [`local-volume-provisioner`](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) 的工作原理。
```
Comment (ran-huang) [inline zh/destroy-a-tidb-cluster.md:55]:
```suggestion
> 为了确保数据安全，在任何情况下都不要删除 PV，除非你熟悉 [`local-volume-provisioner`](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) 的工作原理。
```

File: zh/destroy-a-tidb-cluster.md [hunk 2]
```diff
@@ -50,6 +50,10 @@ helm delete ${cluster_name} --purge
 kubectl delete pvc -n ${namespace} -l app.kubernetes.io/instance=${cluster_name},app.kubernetes.io/managed-by=tidb-operator
 ```
 
+> **警告：**
+>
+> 为了确保数据安全，在任何情况下都不要删除 PV，除非你熟悉 [`local-volume-provisioner`](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) 的工作原理。
```
Comment (DanielZhangQD) [inline zh/destroy-a-tidb-cluster.md:55]: 建议和 L42 放一起吧，另外 PV 不一定是通过 `local-volume-provisioner` 创建的。

File: zh/destroy-a-tidb-cluster.md [hunk 3]
```diff
@@ -43,6 +43,8 @@ helm delete ${cluster_name} --purge
 > **警告：**
 >
 > 下列命令会彻底删除数据，务必考虑清楚再执行。
+>
+> 为了确保数据安全，在任何情况下都不要删除 PV，除非你熟悉 PV 的工作原理。
```
Comment (DanielZhangQD) [inline zh/destroy-a-tidb-cluster.md:47]: https://internal.pingcap.net/jira/browse/USDOC-13 这个 issue 里面提了目前这个说法不够清晰

Comment (weekface) [reply zh/destroy-a-tidb-cluster.md:47]: any suggestion?

Comment (DanielZhangQD) [reply zh/destroy-a-tidb-cluster.md:47]: Let's get this in first and we can refine it later.
I have talked with Kolbe and he will check that issue and give some suggestions.

**Pull request merged**